actions/knowledge/util.ts

```diff
@@ -1,3 +1,9 @@
+export interface FileDetail {
```

Comment:
one more thing that would be helpful is to move this type to the `@/model` directory?